internal/pkg/manager/spod/bindata/spod.go

```diff
@@ -200,6 +208,7 @@ semodule -i /opt/spo-profiles/selinuxd.cil
 						},
 					},
 				},
+				HostPID: true, // required for the log-enricher
```

Comment:
This is uncool

Reply:
Why is this so?

Reply:
The enricher is using the process ID to retrieve the container ID:
https://github.com/kubernetes-sigs/security-profiles-operator/blob/12d2a6a26d06695506c81ebe5122a6d49ede4082/internal/pkg/daemon/enricher/main.go#L73

Reply:
Why don't we keep it on its own container then to reduce the attack vector? The  only that container gets access to the host PIDs while the daemon doesn't. We can enable communication with the daemon via a Unix domain socket or something of the sorts.

Reply:
Like this https://github.com/kubernetes-sigs/security-profiles-operator/pull/469?

Reply:
@saschagrunert you're a magician! :mage_man: that looks quite promising!